pulsar-client-cpp/python/pulsar/schema/definition.py

```diff
@@ -56,15 +57,20 @@ def _get_fields(cls, dct):
 
 class Record(with_metaclass(RecordMeta, object)):
 
-    def __init__(self, default=None, required_default=False, required=False, *args, **kwargs):
+    def __init__(self, default=None, required_default=False, required=False, decode=False, *args, **kwargs):
```

Comment:
It's not easy to understand what is the meaning of `decode=False|True` here. Also these arguments are exposed to the users (eg: marking a field as "required"), do we need to expose this `decode` too?

Reply:
I think this is not necessary, I remove the param `decode`.